src/main/scala/lsc/InstructionSliceTable.scala

```diff
@@ -0,0 +1,70 @@
+package lsc
+
+import boom.common.{BoomBundle, BoomModule, MicroOp}
+import boom.exu.{BrResolutionInfo, BusyResp, CommitSignals}
+import chisel3._
+import chisel3.core.Bundle
+import chisel3.util._
+import freechips.rocketchip.config.Parameters
+
+class IstIO(implicit p: Parameters) extends BoomBundle
+{
+  val mark = Input(new IstMark)
+  val check = Flipped(Vec(coreWidth, DecoupledIO(UInt(vaddrBits.W)))) // use ready bit as reponse
```

Comment:
I think it would be nice with a custom type, ala IstMark, instead of DecoupledIO since its not being used as Ready-Valid

Reply:
I'll create a custom type for this